### PR TITLE
Workspace assistants view + gallery addition for workspace

### LINKED ISSUE #2650: [Ahuna] Admin 'Workspace assistants' page
[Figma](https://www.figma.com/file/QOxHwppG64GtPocpDhS6Y7/Product?type=design&node-id=3146-19109&mode=design&t=NATsjHgMbFZdKwHy-4)

- Mind the visibility change flow [here](https://www.figma.com/file/QOxHwppG64GtPocpDhS6Y7/Product?type=design&node-id=3114-21997&mode=design&t=YFE5Hncm6N3CLqdb-4)

Related [framing](https://github.com/dust-tt/decisions/issues/111#top) and [figma](https://www.figma.com/file/QOxHwppG64GtPocpDhS6Y7/Product?type=design&node-id=2677-10559&mode=design&t=B7nphNLIUPBjCls5-0)

Prototypes of what's going to be done are [here for members](https://www.figma.com/proto/QOxHwppG64GtPocpDhS6Y7/Product?page-id=2677%3A10559&type=design&node-id=2677-10560&viewport=470%2C31%2C0.25&t=L0qImC25TEHwijGx-1&scaling=min-zoom&starting-point-node-id=2677%3A10560) and [here for admins](https://www.figma.com/proto/QOxHwppG64GtPocpDhS6Y7/Product?page-id=2677%3A10559&type=design&node-id=2929-29336&viewport=470%2C31%2C0.25&t=L0qImC25TEHwijGx-1&scaling=min-zoom&starting-point-node-id=2929%3A29336&show-proto-sidebar=1)

### PR DESCRIPTION
**Pull request by spolu**
Fixes: #2650 

Screenshot from 2023-12-15 12-38-34

### CODE REVIEW

File: front/components/assistant/AssistantPreview.tsx [hunk 1]
```diff
@@ -89,16 +98,86 @@ export function AssistantPreview({
...
+          headers: {
+            "Content-Type": "application/json",
+          },
+          body: JSON.stringify(body),
+        }
+      );
+      if (!res.ok) {
+        const data = await res.json();
+        sendNotification({
+          title: `Error addin Assistant`,
```
Comment (philipperolet) [inline front/components/assistant/AssistantPreview.tsx:130]:
```suggestion
          title: `Error adding Assistant`,
```

File: front/lib/api/assistant/configuration.ts [hunk 2]
```diff
@@ -473,7 +474,9 @@ export async function getAgentConfigurations(
   }
 
   if (agentsGetView === "dust") {
-    return (await Promise.all([[], getGlobalAgentConfigurations()])).flat();
+    return (
```
[the remark this answers is not in the document]
Comment (spolu) [reply front/lib/api/assistant/configuration.ts:481]: done

File: front/components/assistant/AssistantPreview.tsx (hunk 1 above)
Comment (spolu) [reply front/components/assistant/AssistantPreview.tsx]: done :+1:

**Review by philipperolet — APPROVED** [review]
:+1: 
modulo build & lint passing

**Pull request merged**